tools/topology/topology1/development/sof-imx8mp-compr-vorbis-wm8960.m4

```diff
@@ -0,0 +1,96 @@
+#
+# Topology with codec_adapter processing component for i.MX8MP
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`sai.m4')
+include(`pcm.m4')
+include(`buffer.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include DSP configuration
+include(`platform/imx/imx8.m4')
+
+
+# Post process setup config
+
+ #codec Post Process setup config
+#
+# Define the pipelines
+#
+# PCM0 <----> volume <-----> SAI3 (wm8960)
+#
+
+
+DECLARE_SOF_RT_UUID("Cadence Codec", cadence_codec_uuid, 0xd8218443, 0x5ff3,
+                    0x4a4c, 0xb3, 0x88, 0x6c, 0xfe, 0x07, 0xb9, 0x56, 0xaa);
+
+define(`CA_UUID', cadence_codec_uuid)
+
+# Post process setup config
```

Comment:
How are these created for vorbis ? This should be in the comment.

Reply:
I've noticed while pushing that the topologies for the other platforms weren't on the main branch either, and that's because that thing isn't fully decided (we're currently using some byte in the configuration data). I'll remove the topology commits from this pull request and instead resubmit them on our local branch.

Reply:
@paulstelian97 that's fine.